src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
```

Comment:
Found the issue causing the messing of styling on the home page here. 
since you have commented out App.js, App.css has to be imported here
`import './App.css'`